<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>801</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>열심히는 필요없다. 잘해라. - 산지직송 대표 -</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEmergency">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>120</y>
     <width>131</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string> EmerGency STOP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSerch">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>131</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>1021</width>
     <height>291</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Goal</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Current</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>At_Work</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>P</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NP</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="editName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editId">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
